mlibrary/lib.umich.edu
# updates

# Overview
Monthly dependency updates to help maintain lib.umich.edu.

These are the August 2024 updates. [JIRA](https://mlit.atlassian.net/jira/software/projects/WEBSITE/boards/27?selectedIssue=WEBSITE-146)

- @eslint/js: 9.9.0
- @stylistic/eslint-plugin: 2.6.2
- eslint: 9.9.0

https://deploy-preview-298--future-wwwlib-previews.netlify.app/

## Testing
```
rm -rf node_modules &&
rm package-lock.json &&
npm install &&
npm run start
```

- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- _How to use the feature_.
